docs/getting-started/server/guide.md

```diff
@@ -303,17 +306,18 @@ repository.
 
     <bitwarden>
 
-    - Copy the user secrets file from the shared Development collection into the `dev` folder.
-    - If you don't have access to the Development collection, contact our IT Manager to arrange
-      access. Make sure you have first set up a Bitwarden account using your company email address.
-    - This `secrets.json` is configured to use the dockerized Azurite and MailCatcher instances and
-      is recommended for this guide.
+    1. Go to Bitwarden's shared Development collection.
+    2. Search for the `Server User Secrets` credentials.
+    3. Download the `secrets.json` file and copy it into the server `dev` folder. (This
+       `secrets.json` file is configured to use the dockerized Azurite and MailCatcher instances and
+       is recommended for this guide.)
```

Comment:
No full stops are needed for numbered lists. (It's probably needed for the last item because of the separate sentence/parens, but that's an exception)

```suggestion
    1. Go to Bitwarden's shared Development collection
    2. Search for the `Server User Secrets` credentials
    3. Download the `secrets.json` file and copy it into the server `dev` folder. (This
       `secrets.json` file is configured to use the dockerized Azurite and MailCatcher instances and
       is recommended for this guide.)
```